content/03.methods.md

```diff
@@ -70,23 +70,24 @@ A summary of processed gender predictions of quotes at each point of processing
 Because we separate first and last authors, we only considered papers with more than one author.
 As for quotes, we needed to extract the first name of the authors.
 We cast names to lowercase and processed them using the R package humaniformat [@https://cran.r-project.org/web/packages/humaniformat/index.html].
-humaniformat identifies if names are reversed (Lastname, Firstname), as well as identifies middle names.
+humaniformat is a rule based program that uses character markers to identify if names are reversed (Lastname, Firstname), find middle names and titles.
 This processing was not required for quote prediction because names written in news articles did not appear to be reversed or abbreviated.
 Since many last or first authorships may be non-names, we additionally filtered out any identified names if they partially or fully match any of the following terms: "consortium", "group", "initiative", "team", "collab", "committee", "center", "program", "author", or "institute".
 Furthermore, since many papers only contain first name initials (for example, "N. Davidson"), we remove any names less than four letters (length includes punctuation) and containing a "." or "-", then strip out all periods from the first name.
 This ensures that hyphenated names are not changed, e.g. Julia-Louise remains unchanged, but removes hyphenated initials, e.g. J-L.
-Finally, we only consider any remaining first names of more than two characters.
-This is to eliminate first and middle jointly-initialized names.
-For example, "NR Davidson" would be reduced to "Davidson" and then eliminated due to the lack of a first name.
+Finally, we only consider any remaining first names of one or more characters.
```

Comment:
so, technically, a name will pass if the first and last name, including a space, is at least 3 characters. So "J L" would pass. I will remove the sentence since this filter only ensures that there is at least one character in the first and last name.

Reply:
removed in https://github.com/greenelab/nature_news_manuscript/pull/14/commits/ee4aca01988a71a5564854ee04bd0b015d8f175c